code/modules/mob/dead/observer/observer.dm

```diff
@@ -206,12 +206,25 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 // This is the ghost's follow verb with an argument
 /mob/dead/observer/proc/ManualFollow(atom/movable/target)
-	if(target && target != src)
-		if(orbiting && orbiting == target)
-			return
+	if (!istype(target))
+		return
+
+	var/icon/I = icon(target.icon,target.icon_state,target.dir)
+
+	var/orbitsize = (I.Width()+I.Height())/2

```

Comment:
We should!

I have doubts those proc names would ever be invalidated (Look at byond's large history of backwards compatabiliy) so in worst case scenario, those proc names return to the same overhead we have these days with Width() and Height().

You can ignore this though if you wan't, it's not major.